### PR TITLE
Partner Review

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project I used HTML,CSS and JS skills

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    });
+    li.appendChild(removeButton);
+    list.appendChild(li);
+  });
+
+  const span = document.querySelectorAll('.text');
+  span.forEach((btn, index) => {
+    btn.addEventListener('keyup', () => {
+      const test = utils.retrieve();
+      test[index].description = btn.value;
```
Comment (deniodev) [inline index.js]: Kindly use descriptive variable names.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/51f27b15da.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="heading">
```
Comment (deniodev) [inline index.html]: Kindly use Semantic Elements in HTML instead of using a div you should use the proper tag for the header.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    margin: 0;
+  }
+  
+  h1 {
+    margin: 0;
+    padding: 0;
+  }
+  
+  .heading {
+    display: flex;
```
Comment (deniodev) [inline style.css]: Kindly use one CSS rule for multiple elements that have the same properties.

**Review by deniodev — CHANGES_REQUESTED** [review]
Hello @tumainimaganiko 👋

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

**Suggested changes**

- [ ] Make sure that the linters work perfectly

Check the comments under the review.

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by deniodev** [comment]
Hi @tumainimaganiko 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉
151541305-67f4f8e4-c79e-4f9d-a6cf-2c7efc3102c6


Highlights
No Linters errors 💯
All requirements are complete 💯
The project is working as expected 💯
Optional suggestions
No optional suggestions :)

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by deniodev — APPROVED** [review]

**Comment by tumainimaganiko** [comment]
Thank you partner @deniodev  for your suggestion to improve my code.

**Pull request merged**